node/impl/full/eth.go

```diff
@@ -640,11 +669,12 @@ func (a *EthModule) EthSendRawTransaction(ctx context.Context, rawTx ethtypes.Et
 		smsg.Message.Method = builtinactors.MethodSend
 	}
 
-	cid, err := a.MpoolAPI.MpoolPush(ctx, smsg)
+	_, err = a.MpoolAPI.MpoolPush(ctx, smsg)
```

Comment:
There's a race condition here. Let's add the entry to the lookup table here synchronously, since the client could ask for its transaction through another JSON-RPC call before the mpool handler has processed the entry.

BTW -- what are we doing to clean up the mapping table from transactions that never get included?